AzureDay.WebApp/AzureDay.WebApp.Notification/Email/Model/RestorePasswordMessage.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace TeamSpark.AzureDay.WebSite.Notification.Email.Model
```

Comment:
the whole file is redundant and should be removed, restore password will be handled by azure ad b2c